react/components/SKUSelector/components/SelectorItem.tsx

```diff
@@ -48,13 +60,16 @@ const SelectorItem: FC<Props> = ({
   showBorders = true,
 }) => {
   const discount = getDiscount(maxPrice, price)
+  const handles = useCssHandles(CSS_HANDLES, {
+    migrationFrom: 'vtex.store-components@3.x',
```

Comment:
Why do we need this?

Reply:
Because I'm not using css modules anymore, so if I don't pass this, the old handles with  `store-component-3-x` won't be added